src/uwtools/drivers/driver.py

```diff
@@ -515,6 +632,7 @@ def _add_docstring(class_: type, omit: Optional[list[str]] = None) -> None:
     :param dry_run: Run in dry-run mode?
     :param key_path: Keys leading through the config to the driver's configuration block.
     :param batch: Run component via the batch system?
+    :param schema_file: Path to schema file to use to validate an external driver.
```

Comment:
@maddenp-noaa I added the `schema_file` parameter here to create the docstring dynamically. I also updated the test case in `test_driver` for the `_add_docstring` method. I didn't see the docstrings get added to each of the classes like I expected it would appear once I run the tests again and pushed to the PR. Is there something else I'm missing to get this to dynamically generate correctly?

Reply:
If this problem persists after the static docstrings in the classes above are removed, we can try to diagnose, but I think that should be done first.

Reply:
I believe I removed all the places where I added the static doc strings. This is the only place I added it for the add_docstring method itself

Reply:
I just added some suggestion/comments showing what I think needs to be removed. Sorry if it wasn't clear previously, and I hope these help.